apps/blueman-sendto.in

```diff
@@ -135,6 +115,19 @@ class SendTo:
             return False
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-d", "--device", dest="device", action="store",
+                        help=_("Send files to this device"), metavar="ADDRESS")
+    parser.add_argument("--dest", dest="device", action="store",
+                        help=_("Same as --device"), metavar="ADDRESS")
+    parser.add_argument("-s", "--source", dest="source", action="store",
+                        help=_("Source adapter. Takes address or adapter's name eg. hci0"), metavar="PATTERN")
+    parser.add_argument("-u", "--delete", dest="delete", action="store_true", help=_("Delete files on exit"))
+    parser.add_argument("files", nargs="*", action="append", metavar="FILE",
```

Comment:
Why `action="append"`? I don't think that has an effect to the usage you're just using the first bunch anyway (`parsed_args.files[0]`).